catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -274,37 +239,20 @@
     /**
      * JAX-RS Server that represents a CSW v2.0.2 Server.
      */
-    public CswEndpoint(BundleContext context, CatalogFramework ddf, FilterBuilder filterBuilder,
-            FilterAdapter filterAdapter, TransformerManager mimeTypeManager,
-            TransformerManager schemaManager, TransformerManager inputManager) {
+    public CswEndpoint(BundleContext context, CatalogFramework ddf,
```

Comment:
IntelliJ shows a bunch of warnings. We may want to take a look and see which ones of those could be addressed.